### PR TITLE
Exclude property slugs

### PR DESCRIPTION
**Pull request by ruttenb**
Allows sites to exclude particular properties from the properties form based on slug.

### CODE REVIEW

File: attachments/templates/attachments/edit_properties.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 <div class="popover">
     <div class="arrow"></div>
-    <form action="{% url "update-attachment" attach_id=att.pk %}" method="post" class="update-attachment">
+    <form action="{% url "update-attachment" attach_id=att.pk %}?{{ querystring }}" method="post" class="update-attachment">
```
Comment (dcwatson) [inline attachments/templates/attachments/edit_properties.html]: Why not use hidden inputs? Mixing GET params in a POST action is weird.

**Comment by dcwatson** [comment]
In fact, why not add an `is_active` boolean to the `Property` model or something, so that you can do this via the admin?

**Comment by ruttenb** [comment]
Ah good idea.. I like it!
Although, I think `is_active` may be a little misleading.  How about, `is_editable` defaulted to True? The idea is to just exclude certain properties from the edit popover form.

**Comment by dcwatson** [comment]
That's fine, I'd much rather make this data-driven than add another setting and pass all that around everywhere.

**Comment by ruttenb** [comment]
Yea I agree. That's a much better implementation.  I will close this PR and open a new one tomorrow since most (if not all) of this code won't be needed.

**Pull request closed without merging**